deploy/2_setup.js

```diff
@@ -1,9 +1,11 @@
 const hre = require('hardhat');
 const airnodeAdmin = require('@api3/airnode-admin');
 const apis = require('../data/apis.json');
+const utils = require('../scripts/utils.js');
```

Comment:
I'm not sure if it makes sense to have two of these small files. The API JSONs and the thing that interpolates the correct API can be in one file.